### ISSUE TITLE
[Question] Are the OG names possible?

### ISSUE THREAD

**Issue by SaltySpaghetto** [opiter09/PVZ-NDS-Manipulator#1]
basically, I'm wondering if the actual file names of the files are possible to decrypt instead of gibberish.

if possible how can it be done?

**Comment by opiter09** [comment]
Nope. Remember, data inside a game is designed to be read by machines, not people.
